src/edu/stanford/dlss/was/WasapiResponseParser.java

```diff
@@ -0,0 +1,16 @@
+package edu.stanford.dlss.was;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class WasapiResponseParser {
+
+  public WasapiResponseParser() { }
+
+  public WasapiResponse parse(byte[] jsonData) throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    WasapiResponse responseObject = objectMapper.readValue(jsonData, WasapiResponse.class);
```

Comment:
Would it be possible to take a java.io.InputStream as an argument to this method and then use ObjectMapper.readValue(InputStream, Class<T>) instead?

http://fasterxml.github.io/jackson-databind/javadoc/2.8/com/fasterxml/jackson/databind/ObjectMapper.html#readValue(java.io.InputStream,%20java.lang.Class)

The reason I am asking for this is that this (seemingly minor) change here will make it quite a bit easier and more efficient on the networking side:

http://stackoverflow.com/questions/17464263/apache-httpclient-most-efficient-way-to-read-response

(i.e. we could call WasapiResponseParser.parse() more easily without having to go via an intermediate byte array first).